lib/timezonedata/php-bc.php

```diff
@@ -16,67 +16,319 @@
+    'America/Nassau',
+    'America/New_York',
+    'America/Nome',
+    'America/Noronha',
+    'America/North_Dakota/Beulah',
+    'America/North_Dakota/Center',
+    'America/North_Dakota/New_Salem',
+    'America/Nuuk',
+    'America/Ojinaga',
+    'America/Panama',
+    'America/Paramaribo',
+    'America/Phoenix',
+    'America/Port-au-Prince',
+    'America/Port_of_Spain',
+    'America/Porto_Velho',
+    'America/Puerto_Rico',
+    'America/Punta_Arenas',
+    'America/Rankin_Inlet',
+    'America/Recife',
+    'America/Regina',
+    'America/Resolute',
+    'America/Rio_Branco',
+    'America/Santarem',
+    'America/Santiago',
+    'America/Santo_Domingo',
+    'America/Sao_Paulo',
+    'America/Scoresbysund',
+    'America/Sitka',
+    'America/St_Barthelemy',
+    'America/St_Johns',
+    'America/St_Kitts',
+    'America/St_Lucia',
+    'America/St_Thomas',
+    'America/St_Vincent',
+    'America/Swift_Current',
+    'America/Tegucigalpa',
+    'America/Thule',
+    'America/Tijuana',
+    'America/Toronto',
+    'America/Tortola',
+    'America/Vancouver',
+    'America/Whitehorse',
+    'America/Winnipeg',
+    'America/Yakutat',
+    'Antarctica/Casey',
+    'Antarctica/Davis',
+    'Antarctica/DumontDUrville',
+    'Antarctica/Macquarie',
+    'Antarctica/Mawson',
+    'Antarctica/McMurdo',
+    'Antarctica/Palmer',
+    'Antarctica/Rothera',
+    'Antarctica/Syowa',
+    'Antarctica/Troll',
+    'Antarctica/Vostok',
+    'Arctic/Longyearbyen',
+    'Asia/Aden',
+    'Asia/Almaty',
+    'Asia/Amman',
+    'Asia/Anadyr',
+    'Asia/Aqtau',
+    'Asia/Aqtobe',
+    'Asia/Ashgabat',
+    'Asia/Atyrau',
+    'Asia/Baghdad',
+    'Asia/Bahrain',
+    'Asia/Baku',
+    'Asia/Bangkok',
+    'Asia/Barnaul',
+    'Asia/Beirut',
+    'Asia/Bishkek',
+    'Asia/Brunei',
     'Asia/Calcutta',
-    'Asia/Chungking',
-    'Asia/Dacca',
-    'Asia/Istanbul',
+    'Asia/Chita',
+    'Asia/Choibalsan',
+    'Asia/Colombo',
+    'Asia/Damascus',
+    'Asia/Dhaka',
+    'Asia/Dili',
+    'Asia/Dubai',
+    'Asia/Dushanbe',
+    'Asia/Famagusta',
+    'Asia/Gaza',
+    'Asia/Hebron',
+    'Asia/Ho_Chi_Minh',
+    'Asia/Hong_Kong',
+    'Asia/Hovd',
+    'Asia/Irkutsk',
+    'Asia/Jakarta',
+    'Asia/Jayapura',
+    'Asia/Jerusalem',
+    'Asia/Kabul',
+    'Asia/Kamchatka',
+    'Asia/Karachi',
+    'Asia/Kathmandu',
     'Asia/Katmandu',
-    'Asia/Macao',
-    'Asia/Saigon',
-    'Asia/Tel_Aviv',
-    'Asia/Thimbu',
-    'Asia/Ujung_Pandang',
-    'Asia/Ulan_Bator',
-    'Atlantic/Faeroe',
-    'Atlantic/Jan_Mayen',
-    'Australia/ACT',
-    'Australia/Canberra',
-    'Australia/LHI',
-    'Australia/North',
-    'Australia/NSW',
-    'Australia/Queensland',
-    'Australia/South',
-    'Australia/Tasmania',
-    'Australia/Victoria',
-    'Australia/West',
-    'Australia/Yancowinna',
-    'Brazil/Acre',
-    'Brazil/DeNoronha',
-    'Brazil/East',
-    'Brazil/West',
-    'Canada/Atlantic',
-    'Canada/Central',
-    'Canada/Eastern',
```

Comment:
The failing unit test happens to use `Canada/Eastern`, which is removed here.
https://www.php.net/manual/en/timezones.others.php
"Warning: Please do not use any of the timezones listed here (besides UTC), they only exist for backward compatible reasons, and may expose erroneous behavior. Furthermore, these timezones may be removed from the IANA timezone database at any time."

And `Canada/Eastern` is listed there.

The unit test can be fixed by using a "proper" timezone in the test. But we need to think and properly understand what happens in genera if we remove these timezones from the data here.